index.html

```diff
@@ -1,52 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="css/style.css">
-  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon-32x32.png">
-  
-  <title>Meet landing page</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./assets/favicon-32x32.png"
+    />
 
-  <header class="relative">
-    <div class="header-logo">
-      <img src="assets/logo.svg" alt="Meet-logo.svg">
-    </div>
-    <div class="header-faces-left"></div>
-    <div class="header-text">
-      <h1>Group Chat for Everyone</h1>
-      <p>Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.</p>
-      <button><a href="#">Download v1.3</a></button>
-      <button><a href="#">What is it?</a></button>
-    </div>
-    <div class="header-faces-right"></div>
-  </header>
-  <main class="relative">
-    <section class="photo-grid">
-      <figure><img src="assets/desktop/image-women-videochatting.jpg" alt=""></figure>
-      <figure><img src="assets/desktop/image-woman-in-videocall.jpg" alt=""></figure>
-      <figure><img src="assets/desktop/image-men-in-meeting.jpg" alt=""></figure>
-      <figure><img src="assets/desktop/image-man-texting.jpg" alt=""></figure>
-    </section>
-    <section class="modern-use">
-      <h4>Built for modern use</h4>
-      <h2>Smarter meetings, all in one place</h2>
-      <p>Send messages, share files, show your screen, and record your meetings — all in one 
-        workspace. Control who can join with invite-only team access, data encryption, and data export.</p>
-    </section>
-  </main>
-  <footer class="relative">
-    <div class="background-footer">
-      <h1>Experience more together</h1>
-      <p>Stay connected with reliable HD meetings and unlimited one-on-one and group video sessions.</p>
-      <button><a href="#">Download v1.3</a></button>
-    </div>
-  </footer>
+    <title>Meet landing page</title>
+  </head>
+  <body>
+    <main>
```

Comment:
Check desktop resolution :
![image](https://user-images.githubusercontent.com/90243506/134236081-aea32d4a-0d9d-474f-b09c-de64aad814d3.png)
Pay attention to the mobile resolution.